### ISSUE TITLE
About sp seismograph Timezone

### ISSUE THREAD

**Issue by abhishek73magar** [crotwell/seisplotjs#38]
hello sir,

while working with seismograph i like to change timezone UTC to local is any functions are available in seismograph config like helicorder timezone set.

thank you !

**Comment by crotwell** [comment]
I will look into this, seems a good idea.

**Comment by crotwell** [comment]
Sorry for long delay, believe this is fixed in commit 15ded4f

In next release you can:
```
    let seisConfig = new sp.seismographconfig.SeismographConfig();
    const ianaTZ = "US/Eastern";
    seisConfig.xAxisTimeZone = ianaTZ;
    seisConfig.xSublabel = ianaTZ;
```
and the x axis for the seismograph will be in US/Eastern instead of UTC. This uses luxon's IANAZone, so most common time zone names should work. Alternatively, you can specify an actual FixedOffsetZone. Default is still UTC.

Similar also works for bottom axis in helicorder, for example:
```
    let heliConfig = new sp.helicorder.HelicorderConfig(timeWindow);
    const ianaTZ = "US/Eastern";
    heliConfig.yLabelTimeZone = ianaTZ;
    heliConfig.xAxisTimeZone = ianaTZ;
    heliConfig.xSublabel = ianaTZ;
```
